ActindoWorkFlow/UserInterface/01_WorkflowElements.md

```diff
@@ -0,0 +1,175 @@
+[!!Workflows](ActindoWorkFlow)
+
+# Workflow elements
+
+![Workflow](/Assets/Screenshots/ActindoWorkFlow/Workflows/Workflow.png "[Workflow]")
+
+A workflow is represented with a number of fixed elements, each of them having specific characteristics. These elements are used to design the workflow, which can be edited as necessary according to customer needs. For detailed information, see [Manage workflows](/ActindoWorkFlow/Operation/01_ManageWorkflows.md).
+
+A workflow runs from left to right, from a start point to an end point, which can be in turn linked to another workflow. A workflow must contain at least one action between the start and the end point, which are special types of places. The actions are linked through places with each other, connecting the corresponding actions input and output ports. All these elements are described more in detail below.
+
+> [Info] The *Workflows* module is linked to different system modules and plugins, such as *Invoicing*, *Email*, *OmniChannels*, *Tasks*, among others. The actions available to create workflows depend on the modules installed in the system.
+
+[comment]: <> (Processes - The data input at the start point, for example a document, goes through the different workflow stages -places and actions- until it reaches the end point. The process is than completed.)
+
+
+
+## Places
+
+![Place](/Assets/Screenshots/ActindoWorkFlow/Workflows/Place.png "[Place]")
+
+A place is a step in the workflow and is represented by circle, usually provided with inward and outward arcs. Places serve as link between two actions, joining an output port from the previous action to the corresponding input port of the following action. Places linking mandatory input/output ports are displayed by default and cannot be deleted. Additional places can be added by clicking on the ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button next to the optional input/output port. For detailed information about the types of input/output ports, see [Input and output ports](#input-and-output-ports).
+
```

Comment:
Add information from gitlab:
- A place can hold 0-n tokens (in the workflow process)
- Every workflow starts with exactly one place and ends with exactly one place -> start point/place and end point/place
- A place always has a data type (sometime more, sometimes less specific) so you have a general idea what kind of data flows through it